bot.js

```diff
@@ -239,8 +239,11 @@ function handleHttpRequest(request, response) {
                 var announceChannel = channels.ss13;
                 if(dataObj.announce_channel) {
                     if(dataObj.announce_channel == 'admin') announceChannel = channels.executivedecisions;
+                    bot.channels.get(announceChannel).sendEmbed(new Discord.RichEmbed({"title":dataObj.ticket_holder,"description":dataObj.announce}));
```

Comment:
Depending on what you're going for this might be enough. But if you're going for an embed you might want to format it a bit more.

Maybe the following instead?

```
{"title": "New ahelp request", "url": "byond://ftl13.com:7777", "color": 13632027, "fields": [{"name": "Ticket Requester", "value": "Name of dude"}, {"name": "Ticket Body", "value": "Announce information"}]}
```
I would recommend at least adding a color attribute to help it stand out.

![image](https://user-images.githubusercontent.com/26130695/41818063-fc9af2de-776c-11e8-8ce0-63c15bf0751c.png)

Reply:
Fuck I did forget the colour oof